### PR TITLE
Add ZeroTier service

### PR DESCRIPTION
**Pull request by proletarius101**
ZeroTier creates secure networks between on-premise, cloud, desktop, and mobile devices.

### CODE REVIEW

File: config/services/zerotier.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (erig0) [inline config/services/zerotier.xml:1]: nit: extra new line

File: config/services/zerotier.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>ZeroTier</short>
+  <description>ZeroTier creates secure networks between on-premise, cloud, desktop, and mobile devices.</description>
+  <port protocol="tcp" port="9991"/>
+  <port protocol="udp" port="9991"/>
+</service>
```
Comment (erig0) [inline config/services/zerotier.xml:8]: nit: missing newline at end of file

**Review by erig0 — COMMENTED** [review]
A couple nits, but looks good otherwise. Normally I'd fix those up myself, but your commit is labeled "unverified" like you meant to sign it.

If you want to sign the commit, then please address the nits yourself. If you don't care about signing, then let me know and I'll fix them up for you and merge.

File: config/services/zerotier.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>ZeroTier</short>
+  <description>ZeroTier creates secure networks between on-premise, cloud, desktop, and mobile devices.</description>
+  <port protocol="tcp" port="9991"/>
+  <port protocol="udp" port="9991"/>
```
Comment (erig0) [inline config/services/zerotier.xml:7]: This is the wrong port! Docs say `9993/udp`. TCP is not used, AFAICS.

https://docs.zerotier.com/zerotier/troubleshooting/

**Review by erig0 — CHANGES_REQUESTED** [review]

File: config/services/zerotier.xml (hunk 3 above)
Comment (proletarius101) [reply config/services/zerotier.xml]: Thanks! Fixed

**Review by erig0 — APPROVED** [review]
Update looks good.

**Pull request merged**